plugin/storage/es/spanstore/writer_test.go

```diff
@@ -178,10 +179,16 @@ func TestSpanWriter_WriteSpan(t *testing.T) {
 				spanExistsService.On("Do", mock.AnythingOfType("*context.emptyCtx")).Return(testCase.spanIndexExists, nil)
 
 				serviceCreateService := &mocks.IndicesCreateService{}
+				serviceMapping = strings.Replace(serviceMapping, "${__NUMBER_OF_SHARDS__}", strconv.FormatInt(w.writer.numShards, 10), 1)
```

Comment:
any better way to test this? right now you're using the same replace logic in the code and in the test to ensure the mapping was replaced.